physionet-django/console/templates/console/complete_credential_applications.html

```diff
@@ -48,25 +61,24 @@
 <!-- comment3 -->
         {% if applications %}
           <div class="table-responsive">
-	          <ol>
-              {% for application in applications %}
-                {% if application.reference_contact_datetime %}
-                  <li class="header" id='application_{{ application.user.email }}' style="color:green">
-                {% else %}
-                  <li class="header" id='application_{{ application.user.email }}' style="color:black;">
-                {% endif %}
-                <a href="#application_{{ application.id }}">{{ application.user.username }}</a>
-                </li>
-              {% endfor %}
-	          </ol>
+	         <ol>
             {% for application in applications %}
-              {% if application.reference_contact_datetime %}
-                {% include "console/complete_application_display.html" with color='green' %}
+              {% if not application.reference_contact_datetime %}
```

Comment:
Don't need to worry for this pull request, but in general I think it is good practice (more readable) to have the positive case first. e.g. 

{% if application.reference_contact_datetime %}
DO X
{% else %}
DO X
{% endif %}